Packs/VaronisSaaS/IncidentFields/incidentfield-Varonis_SaaS_Close_Reason.json

```diff
@@ -22,7 +22,17 @@
     "sla": 0,
     "system": false,
     "threshold": 72,
-    "type": "shortText",
+    "type": "singleSelect",
```

Comment:
Changing type might be a braking change when it comes to old data. Please consider that this isn't part of our best practices and it might require a BC note in the Release notes.

Reply:
breaking changes notes were added